marklogic/documents.py

```diff
@@ -251,3 +310,70 @@ def write(
             headers["Accept"] = "application/json"
 
         return self._session.post("/v1/documents", data=data, headers=headers, **kwargs)
+
+    def _get_multipart_documents_response(
+        self, uris: list[str], categories: list[str], **kwargs
+    ) -> Response:
+        """
+        Constructs and sends a multipart/mixed request to the v1/documents endpoint.
+        """
+        params = kwargs.pop("params", {})
+        params["uri"] = uris
+        params["format"] = "json"  # This refers to the metadata format.
+        if categories:
+            params["category"] = categories
+
+        headers = kwargs.pop("headers", {})
+        headers["Accept"] = "multipart/mixed"
+        return self._session.get(
+            "/v1/documents", params=params, headers=headers, **kwargs
+        )
+
+    def read(
+        self, uris: list[str], categories: list[str] = None, **kwargs
+    ) -> list[Document]:
+        """
+        Read one or many documents via a GET to the endpoint defined at
+        https://docs.marklogic.com/REST/POST/v1/documents .
+
+        :param uris: list of URIs to read.
+        :param categories: optional list of the categories of data to return for each
+        URI. By default, only content will be returned for each URI. See the endpoint
+        documentation for further information.
+        """
+        response = self._get_multipart_documents_response(uris, categories, **kwargs)
```

Comment:
Would it be helpful if to return the response code if the URI isn't found?
What if there's a network or authentication error?
With Python, is it generally assumed that the user of these functions will do their own error handling?